Update semitic.md (#394)
tatianamac/selfdefined: 11ty/definitions/semitic.md

---
title: Semitic
slug: semitic

> flag: _ _

Reviewer: ```suggestion
```
The build is failing because we need to delete the `flag` line.

---
title: Semitic
slug: semitic
defined: true
excerpt: related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa, which include Arabic, Hebrew, Aramaic, and Amharic.
speech: adjective
---

related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa, which include Arabic, Hebrew, Aramaic, and Amharic.

> ## Note

Reviewer: ```suggestion

## Note
```

---
title: Semitic
slug: semitic

> flag: _ _

Reviewer: ```suggestion

```
Flags is an optional property, so we can remove it here entirely.

defined: true

> excerpt: 
>     1. Related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa. There are included Arabic, Hebrew, Aramaic and Amharic.
>     2. Mainly referred to as Arabic and Hebrew

Reviewer: ```suggestion
excerpt: related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa, which include Arabic, Hebrew, Aramaic, and Amharic.
```

speech: adjective

> alt_words:
> - _ _
> - _ _

Reviewer: ```suggestion
```

Since we're not offering alternate words here (an optional property), we can erase all this.

---

> 1. Related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa. There are included Arabic, Hebrew, Aramaic and Amharic.
> 2. Mainly referred to as Arabic and Hebrew.

Reviewer: ```suggestion
related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa, which include Arabic, Hebrew, Aramaic, and Amharic.
```

> 

Reviewer: Here, I'd add a note like this:

```
## Note
While Semitic refers to a broad spectrum of languages, races, and cultures above, it is often mistakenly only used to speak about Arabic and Hebrew.

Reviewer: I would also add a section for Related Words, and add [antisemitic](https://www.selfdefined.app/definitions/antisemitism/) Because you're linking internally, you will only need to link `/definitions/antisemitism`

Reviewer: ```suggestion
## Note
While Semitic refers to a broad spectrum of languages, races, and cultures as described above, it is often mistakenly only used to speak about Arabic and Hebrew.

```
